src/gluon/project/ProjectDetails.ts

```
@@ -120,6 +120,9 @@ export class ListTeamProjects implements HandleCommand<HandlerResult> {
                 };
 
                 return ctx.messageClient.respond(msg);
+            }).catch(() => {
+                // Don't display the error - gluonProjectsWhichBelongToGluonTeam already handles it.
+                return success();
```

It makes me uncomfortable swallowing all errors  😓 but difficult to separate different types. I think gluonProjectsWhichBelongToGluonTeam() should just return the projects or an empty array if there are none. We can fix this another time...

---

Also remember this is just catching an unresolved promise and not necessarily errors in the traditional sense. Personally I think we should try wrap errors (in a QMError base type for example) which our promises can throw and we can then check the error code and act accordingly.

---

One day...